src/main/scala/it/pagopa/interop/notifier/service/converters/AgreementEventsConverter.scala

```diff
@@ -3,58 +3,59 @@ import it.pagopa.interop.agreementmanagement.model.persistence._
 import it.pagopa.interop.commons.queue.message.ProjectableEvent
 import it.pagopa.interop.commons.utils.TypeConversions._
 import it.pagopa.interop.commons.utils.errors.ComponentError
-import it.pagopa.interop.notifier.model.persistence.MessageId
-import it.pagopa.interop.notifier.model.{AgreementEventPayload, DynamoEventPayload}
-import it.pagopa.interop.notifier.service.DynamoService
+import it.pagopa.interop.notifier.model
+import it.pagopa.interop.notifier.model.{AgreementPayload, MessageId, NotificationPayload}
+import it.pagopa.interop.notifier.service.impl.DynamoIndexService
 
 import scala.concurrent.{ExecutionContext, Future}
 
 object AgreementEventsConverter {
 
-  def getMessageId(dynamoService: DynamoService)(implicit
+  def getMessageId(dynamoService: DynamoIndexService)(implicit
     ec: ExecutionContext,
     contexts: Seq[(String, String)]
   ): PartialFunction[ProjectableEvent, Future[MessageId]] = { case e: Event =>
     getMessageIdFromEvent(dynamoService, e)
   }
 
-  private[this] def getMessageIdFromEvent(dynamoService: DynamoService, event: Event)(implicit
+  private[this] def getMessageIdFromEvent(dynamoService: DynamoIndexService, event: Event)(implicit
     ec: ExecutionContext,
     contexts: Seq[(String, String)]
   ): Future[MessageId] = event match {
-    case AgreementAdded(a)                       => Future.successful(MessageId(a.id, a.producerId))
+    case AgreementAdded(a)                       => Future.successful(model.MessageId(a.id, a.producerId))
     case AgreementDeleted(id)                    => id.toFutureUUID.flatMap(getMessageIdFromDynamo(dynamoService))
-    case AgreementUpdated(a)                     => Future.successful(MessageId(a.id, a.producerId))
+    case AgreementUpdated(a)                     => Future.successful(model.MessageId(a.id, a.producerId))
     case AgreementConsumerDocumentAdded(id, _)   => id.toFutureUUID.flatMap(getMessageIdFromDynamo(dynamoService))
     case AgreementConsumerDocumentRemoved(id, _) => id.toFutureUUID.flatMap(getMessageIdFromDynamo(dynamoService))
-    case VerifiedAttributeUpdated(a)             => Future.successful(MessageId(a.id, a.producerId))
-    case AgreementActivated(a)                   => Future.successful(MessageId(a.id, a.producerId))
-    case AgreementSuspended(a)                   => Future.successful(MessageId(a.id, a.producerId))
-    case AgreementDeactivated(a)                 => Future.successful(MessageId(a.id, a.producerId))
+    case VerifiedAttributeUpdated(a)             => Future.successful(model.MessageId(a.id, a.producerId))
+    case AgreementActivated(a)                   => Future.successful(model.MessageId(a.id, a.producerId))
+    case AgreementSuspended(a)                   => Future.successful(model.MessageId(a.id, a.producerId))
+    case AgreementDeactivated(a)                 => Future.successful(model.MessageId(a.id, a.producerId))
```

Comment:
any conflict why here we use `model.`?

Reply:
No, I think it was an effect of IntelliJ auto import.